README.md

```diff
@@ -48,6 +35,18 @@ so-called "HTTP APIs" are usually not.
 ```
 
 ## Usage
+### Quarkus 3.15+
+Quarkus is now part of Quarkiverse, so simply add this to your pom.xml:
```

Comment:
Quarkus is not a part of Quarkiverse, this extension is ;)

Reply:
Good catch. The versioning (and docs) will need some polish after we release first version, I'm not sure if quarkiverse-bom which stores versions of the extensions is included by default in quarkus-parent or it needs to be explicitly added by developers - to be tested later.

Reply:
Fixed